graphql.gemspec

```diff
@@ -28,6 +28,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency "guard-rake"
   s.add_development_dependency "guard-rubocop"
   s.add_development_dependency "listen", "~> 3.0.0"
+  s.add_development_dependency "pry"
+  s.add_development_dependency "pry-stack_explorer"
+  s.add_development_dependency "m"
```

Comment:
Yes. I'll extract this to a different PR once this is ready to go (currently waiting for CI)